ddp-workspace/projects/ddp-angio/src/app/app-routing.module.ts

```diff
@@ -2,249 +2,23 @@ import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 
 import {
-    Auth0CodeCallbackComponent,
-    AuthGuard,
     IrbGuard,
-    BrowserGuard,
-    AdminAuthGuard
 } from 'ddp-sdk';
 
-import {
-    ActivityPageComponent,
-    ActivityComponent,
-    DashboardComponent,
-    LoginLandingComponent,
-    ErrorComponent,
-    StayInformedComponent,
-    PasswordComponent,
-    RedirectToLoginLandingComponent,
-    WorkflowStartActivityComponent,
-    InternationalPatientsComponent,
-    SessionExpiredComponent,
-    ActivityLinkComponent,
-    LovedOneThankYouComponent,
-    RedirectToAuth0LoginComponent,
-    AdminLoginLandingComponent
-} from 'toolkit';
-
-import { AboutUsComponent } from './components/about-us/about-us.component';
 import { DataReleaseComponent } from './components/data-release/data-release.component';
-import { MoreDetailsComponent } from './components/more-details/more-details.component';
-import { PrismComponent } from './components/prism/prism.component';
-import { WelcomeComponent } from './components/welcome/welcome.component';
+import { EndEnrollComponent } from './components/end-enroll/end-enroll.component';
 
 const routes: Routes = [
-    {
```

Comment:
removed ALL route to make sure direct static links doesnt work even when types into the browser